main/integration-tests/support/setup.js

```diff
@@ -137,12 +137,15 @@ class Setup {
     return session;
   }
 
-  async createUserSession({
-    userRole = 'internal-guest',
-    username = this.gen.username(),
-    password = this.gen.password(),
-    projectId = [this.defaults.project.id],
-  } = {}) {
+  async createUserSession(
+    {
+      userRole = 'internal-guest',
+      username = this.gen.username(),
+      password = this.gen.password(),
+      projectId = [this.defaults.project.id],
+    } = {},
+    { includeCreds = false } = {},
```

Comment:
Per our Slack discussion, we can take a different approach to this.

Reply:
Done